frontend/pages/procreg/[step].vue

```diff
@@ -11,7 +17,10 @@ const formConfig = mockVwr;
             <h1>{{ $t("Processing Registry") }}</h1>
         </div>
         <div class="uu-container">
-            <SharedMRForm :form-config="formConfig" />
+            <SharedMRForm
+                :form-config="formConfig"
+                :currentStepSlug="stepSlug(route.params.step)"
```

Comment:
I'm getting a linting warning here: `Attribute ':currentStepSlug' must be hyphenated.eslintvue/attribute-hyphenation`


```suggestion
                :current-step-slug="stepSlug(route.params.step)"
```

Your code still works, but it is conventional to use kebab case, even in Vue templates. (Vue will automatically convert it to camel case).